src/engine/qcommon/q_math.cpp

```diff
@@ -760,31 +783,32 @@ int BoxOnPlaneSide( const vec3_t emins, const vec3_t emaxs, const cplane_t *p )
 	}
 
 	// general case
-	dist[ 0 ] = dist[ 1 ] = 0;
-
-	if ( p->signbits < 8 ) // >= 8: default case is original code (dist[0]=dist[1]=0)
-	{
-		for ( i = 0; i < 3; i++ )
-		{
-			b = ( p->signbits >> i ) & 1;
-			dist[ b ] += p->normal[ i ] * emaxs[ i ];
-			dist[ !b ] += p->normal[ i ] * emins[ i ];
-		}
-	}
-
-	sides = 0;
-
-	if ( dist[ 0 ] >= p->dist )
-	{
-		sides = 1;
-	}
-
-	if ( dist[ 1 ] < p->dist )
-	{
-		sides |= 2;
-	}
-
-	return sides;
+	int index[ 3 ];
+	index[ 0 ] = p->signbits;
+	index[ 1 ] = p->signbits >> 1;
+	index[ 2 ] = p->signbits >> 2;
+
+	index[ 0 ] &= 1;
+	index[ 1 ] &= 1;
+	index[ 2 ] &= 1;
+
+	vec3_t rmins, rmaxs;
+	VectorMultiply( p->normal, emins, rmins );
+	VectorMultiply( p->normal, emaxs, rmaxs );
+
+	float dist[ 2 ] = {};
+	dist[ index[ 0 ] ] += rmaxs[ 0 ];
+	dist[ index[ 1 ] ] += rmaxs[ 1 ];
+	dist[ index[ 2 ] ] += rmaxs[ 2 ];
+
+	dist[ !index[ 0 ] ] += rmins[ 0 ];
+	dist[ !index[ 1 ] ] += rmins[ 1 ];
+	dist[ !index[ 2 ] ] += rmins[ 2 ];
+
+	int bit0 = dist[ 0 ] >= p->dist;
```

Comment:
This could be `>` for consistency with the SSE version

Reply:
What does it change?

Reply:
Also: why is it modified in SSE version? Is testing for equality useless?

Reply:
It doesn't really matter since you can't count on exact float equality, but it enables more culling in the case of exact equality and makes the algorithm the same as the SSE one.

With the SSE one it matters a bit more since in the axis-aligned plane case, exact equality might be somewhat common. Changing `>=` to `>` there made the results better agree with the non-SSE algorithm (and enables more culling).

Reply:
OK.  👍️

I shared that part.